File: solutions/2088-minimum-time-to-type-word-using-special-typewriter/minimum-time-to-type-word-using-special-typewriter.cs
public class Solution {
    public int MinTimeToType(string word) {
        int ans = word.Length;
        int cur = 1;
        foreach (char c in word) {
            int pos = c - 96;
            int diff = Math.Abs(pos - cur);
            ans += Math.Min(diff, 26 - diff);
            cur = pos;
        }
        return ans;
    }
}

File: solutions/2088-minimum-time-to-type-word-using-special-typewriter/minimum-time-to-type-word-using-special-typewriter.java
class Solution {
    public int minTimeToType(String word) {
        int ans = word.length();
        int cur = 1;
        for (int i = 0; i < word.length(); i++) {
            int pos = word.charAt(i) - 96;
            int diff = Math.abs(pos - cur);
            ans += Math.min(diff, 26 - diff);
            cur = pos;
        }
        return ans;
    }
}